editors/vim/Portfile

```diff
@@ -138,51 +138,81 @@ foreach s ${pythons_suffixes} {
     "
 }
 
-variant ruby requires ruby18 description {Compatibility variant, requires +ruby18} {}
-variant ruby18 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby30 description {Enable Ruby scripting} {
+variant ruby requires ruby33 description {Compatibility variant, requires +ruby33} {}
+variant ruby19 conflicts ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
```

Comment:
Would it be possible to solve this with a loop as for the +pythonXY variants? It seems difficult to maintain the conflicting versions like this.